widgets/src/components/TaskList/TaskListContainer.js

```diff
@@ -85,7 +90,24 @@ class TaskList extends React.Component {
         lastPage: taskList.metadata.lastPage === 1,
         size: taskList.metadata.size,
         connection: config.knowledgeSource,
-      });
+      };
+
+      if (!taskList.payload.length) {
+        this.handleDisableScreen('taskList.emptyList');
+      } else {
+        this.setState({
+          ...states,
+          columns: normalizeColumns(
```

Comment:
is it a duplicated code from line 85?

Reply:
sorry, there's a mistake when I resolve the merge. Should be okay now